process/ovpn.cpp

```diff
@@ -204,6 +206,14 @@ int OVPNPlugin::pre_update(Flow &rec, Packet &pkt)
 void OVPNPlugin::pre_export(Flow &rec)
 {
    RecordExtOVPN *vpn_data = (RecordExtOVPN *) rec.get_extension(RecordExtOVPN::REGISTERED_ID);
+
+   //do not export ovpn for short flows, usually port scans
+   uint32_t packets = rec.src_packets + rec.dst_packets;
+   if (packets <= 5) { 
```

Comment:
5 should be probably some defined constant, similarly to c_min_data_packet_size?

Reply:
fixed in following commit